Services/SystemServer/Service.cpp

```diff
@@ -343,6 +348,16 @@ void Service::save_to(JsonObject& json)
     for (String& arg : m_extra_arguments)
         extra_args.append(arg);
     json.set("extra_arguments", move(extra_args));
+
+    JsonArray boot_modes;
+    for (String& mode : m_boot_modes)
+        boot_modes.append(mode);
+    json.set("boot_modes", boot_modes);
+
+    JsonArray environment;
+    for (String& env : m_environment)
+        boot_modes.append(env);
+    json.set("environment", environment);
```

Comment:
:thinking: why is this stuff commented out?

Reply:
Does passing a JsonArray crash the inspector?

Reply:
An array, I think. At least it did when I first wrote that code.

Reply:
I know that an empty array doesn't crash; let me check with an array that has something.

Reply:
It seems to be okay with showing arrays (`[{"pid":25,"command":"te","running_time":5157,"pgid":25,"is_running_in_background":true}]` at least)

Reply:
Why don't you uncomment the code here and try that.

Reply:
Fiiine, I'll pull the PR

Reply:
Seems OK to me.